libs/dh/wholesale/feature-settlement-reports/src/lib/dh-wholesale-settlement-reports.component.html

```diff
@@ -27,6 +27,7 @@ <h3 class="watt-on-light--high-emphasis">
         <dh-wholesale-form
           (filterChange)="onFilterChange($event)"
           [loading]="loading"
+          [executionTime]="executionTime"
```

Comment:
Suggestion: do you think we should do the input as all the filters instead of only the `execution time` for future-proofing?

Reply:
Good suggestion, let's revisit it in the next PR 👍 Would especially be nice if it could avoid the `patchValue` in the input setter, as that felt a little hacky.